bugtool/cmd/root.go

```diff
@@ -382,6 +390,12 @@ func writeCmdToFile(cmdDir, prompt string, k8sPods []string, enableMarkdown bool
 	}
 	// Write prompt as header and the output as body, and / or error but delete empty output.
 	output, err := execCommand(prompt)
+	// log.Print(output)
+	if hashFunc != nil {
+		output = hashFunc(output)
+		log.Print(output)
```

Comment:
```go
if hashFunc != nil {
		output = hashFunc(output)
```
^ This is good

```go
		log.Print(output)
```
^ This looks wrong, you do not have to log the output - it's already implemented later in the function, see

```go
		fmt.Fprint(f, output)
```

In `writeCmdToFile()` I think you just have to post-process the `ouptut` with `output = hashFunc(output)`, and let the function handle the resulting string for the logging part.